Formula/i2pd.rb

```diff
@@ -0,0 +1,91 @@
+class I2pd < Formula
+  desc "Full-featured C++ implementation of I2P client"
+  homepage "https://i2pd.website/"
+  url "https://github.com/PurpleI2P/i2pd/archive/2.20.0.tar.gz"
```

Comment:
I noticed the software is using a specially crafted Makefile for Homebrew with hardcoded links . https://github.com/PurpleI2P/i2pd/blob/openssl/Makefile.homebrew - wouldn't it be better to use a generic makefile since the assumptions made in it might not be true for all systems ?

Reply:
It used only when GCC `-dumpmachine` returns `darwin` and `HOMEBREW` is defined.

https://github.com/PurpleI2P/i2pd/blob/35407125171ed46acceb0397680ecd69c233dd9f/Makefile#L35-L37

Reply:
I meant homebrew installs on osx since not all installations install under the default prefix.

Reply:
ah... okay, I will add usage of [BREWROOT](https://github.com/PurpleI2P/i2pd/blob/97ae2674dcdda7ef5acf10eb224f010dab4a21c0/Makefile.homebrew#L2) define...